### PR TITLE
Add CODEOWNER file

### PR DESCRIPTION
**Pull request by dhmlau**
Add CODEOWNER file

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners,
+# the last matching pattern has the most precendence.
+
+# Core team members from IBM
+* @bajtos
```
Comment (ssh24) [inline CODEOWNERS]: Need a new line here.

**Comment by kjdelisle** [comment]
@dhmlau Would you be able to pick this one up again? I'm not getting pings for PRs/issues on this repository and this would help with that. :D

**Comment by dhmlau** [comment]
sure.  Do you want to leave it as-is and adding you, or break it down into `services/*` for different owners?  We could always use the former as a start.

**Comment by kjdelisle** [comment]
I'd add all of us, and not divide it up; we should all be able to work on these examples. If not, then it's a good area to start learning.

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners,
+# the last matching pattern has the most precendence.
```
Comment (loay) [inline CODEOWNERS]: precedence

**Pull request merged**